HW02_B_Mimic.md

```diff
@@ -1,89 +1,128 @@
+make from scratch. I will only provide the MD, not the actual code I
+used to create it besides which data I use to create it. The rest will
+be up to you.
+
+Try for all 4, but if you are struggling don’t worry about it. Try your
+best for each, if you don’t get everything that’s what the peer-review
+is for. :smile:
+
+### Graph 1
+
+``` r
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+Using the diamonds dataset, make this graph:
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
+
+``` r
+ggplot(diamonds, aes(x = cut, color = clarity, fill = clarity))+
+  geom_bar(position = "dodge") +
+  annotate('rect', xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, fill = 'black', alpha = 0.25) +
+  annotate('text', x = 4, y = 4500, label = 'My Best Diamonds,\nof course') +
+  labs(x = "Diamond Cut", 
+       y = "Number of Diamonds", 
+       title = "My Diamond Collection", 
+       subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(hjust = 0.5, face = "bold"))
+```
+### Graph 2
+
+``` r
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+
+    ## `geom_smooth()` using formula 'y ~ x'
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-4-1.png)<!-- -->
+
+``` r
+ggplot(iris, aes(x = Sepal.Length, y = Petal.Length, shape = Species, group = Species, color = Species)) +
+  geom_point() +
+  geom_smooth(se = F, method = lm, color = "black")
+```
+### Graph 3
+
+You’ll need the information in this first box to create the graph
+
+``` r
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+``` r
+
+ggplot(mpg,aes(x = displ, y = hwy))+
+  geom_point()+
+  labs(title = 'Corvettes are a bit of an outlier')
+
+```
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-6-1.png)<!-- -->
+
+There is a trick to getting the model and year to print off together.
+`paste()` is a useful function for this, also pasting together parts of
+file names and parts of urls together.
+
+### Graph 4
+
+``` r
+data(mpg)
+
+#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
+library(RColorBrewer)
+display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
+```
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-7-1.png)<!-- -->
+
+The above graph lets you see some colobrlind friendly palettes. For the
+graph below, I used Set2.
+
+Now using the above mpg dataset, make this graph
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-8-1.png)<!-- -->
+``` r
+palette(Set2)
+ggplot(mpg, aes(x = class, y = cty, color = class)) +
+  geom_boxplot(color = 'black', size = 0.2, outlier.alpha = 0) +
+  scale_color_manual(values = brewer.pal(n = 7, name = "Set2")) +
```

Comment:
I'm curious, what does n=7 mean in this case? I called the color palette in a slightly different way and was just trying to understand your method of doing it!